packages/react/src/components/Tab/Tab-test.js

```diff
@@ -110,6 +110,19 @@ describe('Tab', () => {
       });
     });
   });
+
+    it('should set aria-disabled to the value of the disabled prop', () => {
```

Comment:
Would it be better to render it without the disabled prop at all, ensure that it can't find the element, then find the element using the attribute once the prop has been set?

That way we're also testing the case where no prop is provided.

Reply:
I'm really new to testing so bear with me, how do I check that it can't find the element? In my head it would be something like `expect(getDisabledRegion().prop('aria-disabled')).toBe(undefined);` with disabled not set yet.

Reply:
Nevermind, figured it out!